backend/director/agents/videodb_helper.py

```
@@ -0,0 +1,109 @@
+import logging
+import json
+import re
```

Ruff (0.8.2)

`re` imported but unused

Remove unused import: `re`

(F401)